Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -11,7 +11,7 @@
 //  Measures the performance of a block of code and reports the results.
 //
 
-#if os(Linux) || os(FreeBSD)
+#if USE_OBJC_FOUNDATION || os(Linux) || os(FreeBSD)
```

Comment:
I thought ObjC `Foundation` exist for Linux. So just curious to introduce better name what kind of foundation we use for Linux / FreeBSD.